helm/alfresco-identity-service/values.yaml

```diff
@@ -43,7 +43,11 @@ keycloak:
         mountPath: /opt/jboss/keycloak/themes/alfresco  
     extraArgs: "-Dkeycloak.import=/realm/alfresco-realm.json"
     persistence:
+      deployPostgres: true
+      dbVendor: postgres
       dbPassword: keycloak
+  persistence:
```

Comment:
I think this is a duplicate of the above lines

Reply:
It is not duplicated. It is needed for both of them, since the deploy uses sometimes keycloak.persistence.deployPostgres and sometimes persistence.deployPostgres. Tried to remove one of them and postgres is not deploying anymore with succes.

Reply:
In searching the source I don't see anywhere that `persistence.deployPostgres` is used by the Keycloak chart, only `keycloak.persistence.deployPostgres`.

Their default values and [Postgres override for tests](https://github.com/helm/charts/blob/master/stable/keycloak/ci/postgres-ha-values.yaml) also only reference `keycloak.persistence.deployPostgres` so I think we should investigate why the other is needed, it may be a mistake on our part.

Reply:
Removing persistence.deployPostgres=true from values.yaml will not deploy the identity service with this error:
`Error: render error in "alfresco-identity-service/charts/keycloak/templates/statefulset.yaml": template: alfresco-identity-service/charts/keycloak/templates/_helpers.tpl:39:12: executing "keycloak.postgresql.fullname" at <{{template "postgres...>: template "postgresql.fullname" not defined
`
After some research along with Sergiu, we discovered that setting `persistence.deployPostgres=true` will make the error to disappear and the deploy to work properly.
I tried now to move the section 
```
persistence:
    deployPostgres: true
    dbVendor: postgres
    dbPassword: keycloak
```
outside keycloak and seems to work fine; no need for duplicate. I will do some more tests to check if the deployement works fine.

Reply:
Thanks Ramona when you are ready to have me verify the issue with the latest change pls signal me